Gems/ROS2/Code/Source/Lidar/ROS2Lidar2DSensorComponent.h

```diff
@@ -73,8 +73,7 @@ namespace ROS2
 
         RaycastResult m_lastScanResults;
 
-        AZ::u32 m_ignoredLayerIndex = 0;
-        bool m_ignoreLayer = false;
+        AZStd::vector<AZ::u32> m_ignoredLayerIndices = {};
```

Comment:
suggestion: rename both to m_ingoredCollisionLayers, use an unordered set, unless UX for the Editor is better with an ordered one.